modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -482,6 +482,162 @@ then put them into a `Background:` section. For example:
 
 This reduces some duplication in feature files.
 
+[[controlling-test-scenarios]]
+== Controlling Running Test Scenarios In Different Environments
+
+A feature or test scenario might only be relevant to run on a system-under-test
+that has a particular environment. For example, a particular app enabled.
+
+To allow the test runner script to run the features and scenarios relevant to
+the system-under-test the feature file or individual scenarios are tagged.
+The test runner script can then filter by tags to select the relevant features and/or scenarios.
+
+=== Tagging A Whole Feature Or Single Scenario
```

Comment:
I would delete that paragraph, that is gherkin knowledge, not owncloud

Reply:
Removed and replaced with a link to the Behat docs about tags.